### DISCUSSION TITLE
Can someone explain to me these opentelemetry traces?

### DISCUSSION THREAD

**Discussion by williambrach** [fastapi/fastapi#9233] [Questions]
### First Check

- [X] I added a very descriptive title here.
- [X] I used the GitHub search to find a similar question and didn't find it.
- [X] I searched the FastAPI documentation, with the integrated search.
- [X] I already searched in Google "How to X in FastAPI" and didn't find any information.
- [X] I already read and followed all the tutorial in the docs and didn't find an answer.
- [X] I already checked if it is not related to FastAPI but to [Pydantic](https://github.com/pydantic/pydantic).
- [X] I already checked if it is not related to FastAPI but to [Swagger UI](https://github.com/swagger-api/swagger-ui).
- [X] I already checked if it is not related to FastAPI but to [ReDoc](https://github.com/Redocly/redoc).

### Commit to Help

- [X] I commit to help with one of those options 👆

### Example Code

```python
No code is required.
```

### Description

Hi, 

I am currently attempting to load-test my API and have discovered that it runs very slowly during high-load situations. To address this issue, I have implemented open telemetry using the Kubernetes operator with python auto instrumentation.

Attached below is a screenshot of one of the traces. As you can see, there is a significant delay between the incoming request being received and the code being executed. I am curious if someone could explain to me why this is happening. My hypothesis is that the fast API worker is unable to handle the incoming requests due to the high volume of traffic, resulting in a backlog of requests in the queue. I am also attaching a request from low demand situation. 

High-load situation
<img width="813" alt="image" src="https://user-images.githubusercontent.com/29985921/224121038-57668896-e5da-4662-a1fe-de3846d10c93.png">

Low-load situation
<img width="797" alt="image (1)" src="https://user-images.githubusercontent.com/29985921/224121051-e5dd955e-b8ff-49ef-847e-ccce88010ce8.png">

Thank you.

### Operating System

Linux, Other

### Operating System Details

An issue was discovered at the AKS cluster hosted on Microsoft Azure. 

### FastAPI Version

0.68.0

### Python Version

Python 3.8.12

**Comment by sudoTJ** [comment]
Did you ever solve this to get the missing telemetry? I am facing somewhat of the same issue

**Reply by williambrach** [reply]
sadly no.
